### PR TITLE
Feature/character selection two

### PR DESCRIPTION
**Pull request by m4rtint**
- Making sure reset works
- Making sure soft reset works. 
- Video is on slack

### CODE REVIEW

**Comment by Gursaanj** [comment]
[Implement new design for player selection](https://trello.com/c/xehnrwL7/353-implement-new-design-for-player-selection)

File: Assets/Script/ScriptableObject/CarChoices.cs [hunk 1]
```diff
@@ -24,6 +24,12 @@ public class CarChoices : ScriptableObject
     [PreviewField(Height = 100)]
     [TableColumnWidth(500)]
     [SerializeField]
+    private Material m_TakenCarMaterial;
```
Comment (Gursaanj) [inline Assets/Script/ScriptableObject/CarChoices.cs]: should be m_takenCarMaterial

**Comment by Gursaanj** [comment]
@m4rtint  please upload a gif, set of screenshots if possible. Thanks

File: Assets/Script/UI/MainMenu/ControllerSelection.cs [hunk 2]
```diff
@@ -84,23 +77,33 @@ private void OnReset()
...
     private void OnEnable()
     {
-        AddDelegate();
+        OnReset();
         m_canPlayersJoin = true;
     }
 
     private void OnDisable()
     {
-        RemoveDelegate();
```
Comment (m4rtint) [inline Assets/Script/UI/MainMenu/ControllerSelection.cs]: Note to self - need to add RemoveDelegate OnDestroy()

**Pull request merged**